src/createStore.js

```diff
@@ -22,8 +24,8 @@ export default ({
     initialState,
     compose(
       applyMiddleware(thunk, ...middleware),
-      ...enhancers,
-      devtools
+      devtools,
+      ...enhancers
```

Comment:
@gaearon Nice catch - I'll play around with that case. This is all leading to a `plugins` option which looks like:

```js
plugins: [router, localStorage, devtools]
```

Plugins can expose any combination of reducers, enhancers, and middleware and they get wired up for you. Ordering is an interesting problem, will look into it.